tools/python/maps_generator/README.md

```diff
@@ -26,35 +23,44 @@ git checkout release-92
 
 The app version can be found in the "About" section in the settings menu of OMaps.
 
-1. [Build and install generator_tool.](https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md#desktop-app)
-2. Change directory:
+2. Build and install the generator_tool.
+
+```sh
+./tools/unix/build_omim.sh -r desktop
+./tools/unix/build_omim.sh -r generator_tool
+./tools/unix/build_omim.sh -r world_roads_builder_tool
+```
+
+3. Change directory:
 
 ```sh
 $ cd omim/tools/python/maps_generator
 ```
 
-3. Install dependencies:
+1. Install dependencies:
 
 ```sh
 maps_generator$ pip3 install -r requirements_dev.txt
 ```
 
-4. Make ini file:
+1 Make the ini configuration file:
```

Comment:
Why 1. ?

Did you preview it? Was the markdown with numbered items rendered correctly?

Reply:
Whoops, fixing.